easybuild/tools/build_log.py

```diff
@@ -103,25 +112,44 @@ def deprecated(self, msg, max_ver):
 
     def nosupport(self, msg, ver):
         """Print error message for no longer supported behaviour, and raise an EasyBuildError."""
-        self.error("NO LONGER SUPPORTED since v%s: %s; see %s for more information" % (ver, msg, DEPRECATED_DOC_URL))
+        nosupport_msg = "NO LONGER SUPPORTED since v%s: %s; see %s for more information"
+        raise EasyBuildError(nosupport_msg, ver, msg, DEPRECATED_DOC_URL)
 
     def error(self, msg, *args, **kwargs):
         """Print error message and raise an EasyBuildError."""
-        newMsg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), msg)
-        fancylogger.FancyLogger.error(self, newMsg, *args, **kwargs)
+        ebmsg = "EasyBuild crashed with an error %s: " + msg
+        args = (self.caller_info(),) + args
```

Comment:
`args` is already a tuple, see also https://docs.python.org/2/tutorial/controlflow.html#arbitrary-argument-lists

`*args` is just a way to refer to the rest of the arguments, there's no tuple reference passed down (and tuples are immutable anyway, other than lists), so no, the 'original' `args` is not modified